src/sections/rich-text.liquid

```diff
@@ -0,0 +1,40 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add "Rich text" section on the Theme Editor.
+
+  Theme Store required settings
+  - Heading: title of the rich text section
+  - Text: description of the rich text section
+{%- endcomment -%}
+<section>
+  {%- if section.settings.title != blank -%}
+    <h2 class="h3">{{ section.settings.title | escape }}</h2>
```

Comment:
Is class "h3" necessary?

Reply:
Definitely not, thanks for pointing this out.